modules/install-sno-installing-with-the-assisted-installer.adoc

```diff
@@ -8,18 +8,22 @@
 
 Use the Assisted Installer to install the single-node cluster.
 
+.Prerequisites
+
+* Ensure that the boot drive order in the server BIOS settings defaults to booting the server from the target installation disk.
+
 .Procedure
 
-. Attach the {op-system} discovery ISO to the target host.
+. Attach the {op-system-first} discovery ISO image to the target host.
 
-. Configure the boot drive order in the server BIOS settings to boot from the attached discovery ISO and then reboot the server.
+. Configure the server to boot from the {op-system} discovery ISO image. The discovery ISO image writes the {op-system} system configuration to the target installation disk and automatically triggers a server restart.
 
 . On the administration host, return to the browser. Wait for the host to appear in the list of discovered hosts. If necessary, reload the link:https://console.redhat.com/openshift/assisted-installer/clusters[*Assisted Clusters*] page and select the cluster name.
 
 . Complete the install wizard steps. Add networking details, including a subnet from the available subnets. Add the SSH public key if necessary.
 
 . Monitor the installation's progress. Watch the cluster events. After the installation process finishes writing the operating system image to the server's hard disk, the server restarts.
 
-. Remove the discovery ISO, and reset the server to boot from the installation drive.
+. Remove the discovery ISO image and reset the server to boot from the installation drive.
```

Comment:
The automatic server restart default to booting from the install disk means you don't need to reset the server.

I would just leave this as:
`Optional. Remove the discovery ISO image.`

Reply:
Thanks. I removed " and reset the server to boot from the installation drive."